models/migrations/migrations.go

```diff
@@ -413,6 +413,10 @@ var migrations = []Migration{
 	NewMigration("Add badges to users", createUserBadgesTable),
 	// v225 -> v226
 	NewMigration("Alter gpg_key/public_key content TEXT fields to MEDIUMTEXT", alterPublicGPGKeyContentFieldsToMediumText),
+	// v226 -> v227
+	NewMigration("Add header_authorization_encrypted column to webhook table", addHeaderAuthorizationEncryptedColWebhook),
+	// v227 -> v228
+	NewMigration("Encrypt the authorization header of matrix webhooks", encryptMatrixWebookAuthorizationHeader),
```

Comment:
Why not merge the two migration as one?

Reply:
_This is my 3rd PR, the first with a migration_

I developed the 2 migrations one after the other and I thought that it would be nicer for them to be split (since they are quite different in scope). Moreover the first one is quite trivial, whereas the second is subtle (and hence tested).

Do you prefer that I merge the two migrations together? (I have no strong feeling and not a seasoned gitea-contributor)

Reply:
ok, I just merged them!